pkg/queue/sharedmain/main.go

```diff
@@ -164,6 +165,12 @@ type Defaults struct {
 	Transport http.RoundTripper
 }
 
+type service struct {
```

Comment:
Nit: this is named service, the methods are called `buildServer` but those return `services`.

Reply:
We are referring to them as services all around, apart from the time they are built. Each service is a service created using a single server. I am not sure we should change that...
 
It will be more confusing to call it `type server struct`

Reply:
Yeah, naming stuff. What about `namedServer`? I'm also fine with keeping it `service`.

Reply:
funcs like `buildAdminServer` return a service which is inconsistent. So I suggest we either say buildAdminService or keep the func name and return something similar to what @ReToCode mentions.

Reply:
Aligned all to service - aligned all build* functions

Reply:
Aligned all to service - aligned all build* functions

Reply:
changed to httpServer everywhere.